template/auth/content/model/user.ts

```diff
@@ -0,0 +1,9 @@
+interface User {
+    fullname?: string;
+    email: string;
+    password?: string;
+}
+
+export {
+    User as default,
```

Comment:
Is there a specific reason why this Interface is exported like this? If not, best to follow how the other interfaces are exported in the models folder, for consistency.